roles/test-utils/sv1-mining-device/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "sv1-mining-device"
+name = "sv1_mining_device"
 version = "0.1.0"
 authors = ["The Stratum V2 Developers"]
-edition = "2021"
+edition = "2018"
```

Comment:
Changed this to keep it consistent with all of the other roles, we should upgrade all of them to 2021 at some point